tests/flow/test_prev_rdb_decode.py

```
@@ -1,12 +1,267 @@
+        decoder_id = 11
+        key = graph_id(decoder_id)
+        rdb = get_dump(decoder_id)
+
+        # restore dump
+        self.redis_con.restore(key, 0, rdb, True)
+
+        # select graph
+        graph = self.db.select_graph(key)
+
+        # expected entities
+        node0  = Node(node_id=0, labels='L1', properties={'val': 1, 'strval': 'str', 'numval': 5.5, 'boolval': True, 'array': [1,2,3], 'point': {'latitude': 32, 'longitude': 34}})
```

Ruff

`Node` may be undefined, or defined from star imports

(F405)